source/retryable-writes/tests/README.rst

```diff
@@ -91,6 +91,11 @@ disabled like so::
         mode: "off"
     });
 
+Speeding Up Tests
+=================
+
+See `Speeding Up Tests </source/transactions/tests/README.rst#speeding-up-tests>`_ in the Transactions spec tests.
```

Comment:
The retryable reads spec already has the most verbose explanation. Please link to that instead.

Also, the CWD of this file is `source/retryable-writes/tests/`, so the URL will likely need to be `../../retryable-reads/tests/README.rst#speeding-up-tests`.